pkg/plugin/response.go

```diff
@@ -98,6 +98,7 @@ func parseStreamResponse(reader io.Reader) backend.DataResponse {
 			return newResponseError(err, backend.StatusInternal)
 		}
 		labelsField.Append(d)
+		labels = data.Labels{}
```

Comment:
It is better to move L:44 to L:85 - define the map before you use it in the loop. Not outside the loop.

Reply:
fixed